flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/consumer/LocalInputChannelTest.java

```diff
@@ -431,8 +426,35 @@ public void testGetNextAfterPartitionReleased() throws Exception {
 		assertFalse(channel.getNextBuffer().isPresent());
 	}
 
+	/**
+	 * Verifies that buffer is not compressed when getting from a {@link LocalInputChannel}.
```

Comment:
remove `not`

Reply:
Buffer from local channel is not compressed.